### ISSUE TITLE
Staging Review finding: The "Filter and sort" link on mobile/at high zoom has redundant `aria-describedby` text.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93691]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User encounters unnecessary repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On a mobile device or at high zoom, a screen reader user who encounters the "Filter and sort" link may hear the link text announced multiple times.

The link includes both visible link text and `aria-describedby` text that each say "Filter and sort." `aria-describedby` is used to provide supplementary information or context to help understand an interaction when the accessible name of the element doesn't include sufficient context. Providing `aria-describedby` text that exactly matches the actual text of the link just results in that text being announced twice.

<details><summary>Link, screenshot or steps to recreate</summary>

The "Filter and sort" link is coded as:
```
<va-link-action href="#" message-aria-describedby="Filter and sort" text="Filter and sort" type="secondary" class="hydrated"></va-link-action>
```

![Portion of page at high zoom with heading "Benefits to explore," an action link with text "Filter and sort," and a bold line of text describing the results currently on the screen.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/82863-Brian-1.png?raw=true)

</details>

## Recommended action

Remove the `message-aria-describedby` attribute.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by nkontrabecki** [comment]
Confirmed and tested that the "Remove the message-aria-describedby attribute." is removed in Sprint 14.
